store/s3.go

```diff
@@ -45,7 +45,7 @@ func NewS3Store(conf *config.Config) (Storage, error) {
 
 // UploadFile path is directory, name is the file name.
 // It is stored in the 'data-market' bucket
-func (s AWSS3Storage) UploadFile(path string, name string, data []byte) error {
+func (s AWSS3Storage) UploadFile(path string, round string, name string, data []byte) error {
```

Comment:
I think the name round is not appropriate because it is also used in dataDeal.
Umm. remove parameter to round, And It looks like you need to specify the round information in the path as well.

Reply:
Ah okay, did you mean specify the path in `validate_data_handler.go`?

Reply:
Fixed in [b59d5f3](https://github.com/medibloc/panacea-data-market-validator/pull/73/commits/b59d5f34145b5f67059ddbfef84483cdb23022b7).